.github/workflows/release.yml

```diff
@@ -58,76 +58,75 @@ jobs:
 
     - name: Cross-compile armv7
       run: |
-        cross build --release --target armv7-unknown-linux-gnueabihf --no-default-features --features notan/glsl-to-spirv
+        cross build --release --target armv7-unknown-linux-gnueabihf --no-default-features --features notan_glsl-to-spirv
         zip -r oculante_armv7_minimal.zip target/armv7-unknown-linux-gnueabihf/release/oculante
       if: matrix.os == 'ubuntu-latest'
 
     # - name: Cross-compile arm64
     #   run: |
-    #     cross build --release --target aarch64-unknown-linux-gnu --no-default-features --features notan/glsl-to-spirv
+    #     cross build --release --target aarch64-unknown-linux-gnu --no-default-features --features notan/shaderc
     #     zip -r oculante_aarch64_minimal.zip target/aarch64-unknown-linux-gnu/release/oculante
     #   if: matrix.os == 'ubuntu-latest'
 
     - name: Install cargo bundle
       run: cargo install cargo-bundle
-      if: matrix.os == 'macos-12'
-      
+      if: matrix.os == 'macos-15-intel' || matrix.os == 'macos-latest'
+
     - name: Install shaderc
       run: brew install shaderc
-      if: matrix.os == 'macos-14'
-
+      if: matrix.os == 'macos-15-intel' || matrix.os == 'macos-latest'
       
     - name: Install cargo deb
       run: cargo install cargo-deb
-      if: matrix.os == 'ubuntu-20.04'
```

Comment:
in general, we might get complaints of the resulting binaries are used by users on older systems